images/build/installNpm.sh

```diff
@@ -5,7 +5,13 @@ set -ex
 for ver in `ls /opt/nodejs`
 do
     nodeModulesDir="/opt/nodejs/$ver/lib/node_modules"
-    npm_ver=`jq -r .version $nodeModulesDir/npm/package.json`
+    packageJson="$nodeModulesDir/npm/package.json"
+
+    if [ ! -f "$packageJson" ]; then
```

Comment:
just curious .. what failure are we having for not having this check earlier?

Reply:
This was a new failure that I saw the day I was trying to get a CI build go through. The failure is below:

```
+ for ver in `ls /opt/nodejs`
+ nodeModulesDir=/opt/nodejs/6.6.0/lib/node_modules
[0m[91m++ jq -r .version /opt/nodejs/6.6.0/lib/node_modules/npm/package.json
[0m[91m+ npm_ver=3.10.3
[0m[91m+ IFS=.
+ read -ra SPLIT_VERSION
[0m[91m+ major=3
+ minor=10
[0m[91m+ '[' 3 == 6 ']'
[0m[91m+ for ver in `ls /opt/nodejs`
[0m[91m+ nodeModulesDir=/opt/nodejs/6.9.3/lib/node_modules
[0m[91m++ jq -r .version /opt/nodejs/6.9.3/lib/node_modules/npm/package.json
[0m[91mjq: error: Could not open file /opt/nodejs/6.9.3/lib/node_modules/npm/package.json: No such file or directory
[0m[91m+ npm_ver=
[0mThe command '/bin/sh -c /tmp/scripts/installNpm.sh' returned a non-zero code: 2
```

From https://devdiv.visualstudio.com/DevDiv/_build/results?buildId=3279541